integrations/gcp/gcp_core/utils.py

```diff
@@ -0,0 +1,48 @@
+import base64
+import json
+from collections.abc import MutableSequence
+from typing import Any, TypeVar
+
+import proto  # type: ignore
+from fastapi import Request
+from port_ocean.context.ocean import ocean
+
+from gcp_core.types import CloudAssetInventoryFeed, SubscriptionMessage
+
+T = TypeVar("T", bound=proto.Message)
+
+
+class GotFeedCreatedSuccessfullyMessage(Exception):
+    pass
+
+
+def parseProtobufMessages(messages: MutableSequence[T]) -> list[dict[str, Any]]:
+    return [proto.Message.to_dict(message) for message in messages]
+
+
+async def parseSubscriptionMessageFromRequest(request: Request) -> SubscriptionMessage:
+    request_json = await request.json()
+    message_id = request_json["message"]["messageId"]
+    try:
+        data = json.loads(base64.b64decode(request_json["message"]["data"]))
+    except json.JSONDecodeError:
+        if (
+            base64.b64decode(request_json["message"]["data"])
+            .decode("utf-8")
+            .startswith("You have successfully configured real time feed ")
+        ):
+            raise GotFeedCreatedSuccessfullyMessage()
+    asset_type = data["asset"]["assetType"]
+    asset_name = data["asset"]["name"]
```

Comment:
is that consistent? can we rely on the existent of those keys? can you add docs reference to the "event" struct example